tests/test_heat.py

```diff
@@ -169,16 +173,15 @@ def _get_min_max_pressure(case):
         min_, max_ = _get_min_max_pressure(case_min_max_pressure)
         self.assertGreater(min_, self.min_pressure * 0.99)
         self.assertLess(max_, self.max_pressure * 1.01)
-        self.assertGreater(case_min_max_pressure.objective_value, base_objective_value * 1.5)
```

Comment:
why was this removed and not replaced by self.assertGreater(case_min_max_pressure.objective_value, base_objective_value)
or add check that the demand is thus not matched, basically what this objective_value should be.

Reply:
I thought this was a bit of a unnecessary check, but I made something to replace it. Please note that both problems do not match demand.